<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>MultiItem</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="lineColorCheck">
            <property name="text">
             <string>Line/Border Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="patternColorCheck">
            <property name="text">
             <string>Pattern Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="lineStyleCheck">
            <property name="text">
             <string>Line Style</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="bubbleStyleCheck">
            <property name="text">
             <string>Bubble Style</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="lineColorTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTableWidget" name="patternColorTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTableWidget" name="bubbleStyleTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QTableWidget" name="lineStyleTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="demoLayout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineColorCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineColorTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternColorCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>patternColorTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bubbleStyleCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>bubbleStyleTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineStyleCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineStyleTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
